extensions/python/pythonprocessors/nifi_python_processors/utils/inline_dependency_installer.py

```diff
@@ -0,0 +1,43 @@
+import ast
+import sys
+import subprocess
+import os
+
+
+class Visitor(ast.NodeVisitor):
```

Comment:
There is a lot of logic and nesting going on here. It would be nice to add a code comment describing what this does, and how it's (meant to be) used.

Reply:
Good point, added some comments in 327ddca15c6bf2f9692d3da061f5688596537c06